### DISCUSSION TITLE
目录下边怎么添加内容，一直没找到，谢谢各位！

### DISCUSSION THREAD

**Discussion by qijixiu** [AlistGo/alist#3995] [Q&A]
![Inked2023-03-30_165633_LI](https://user-images.githubusercontent.com/84918228/228784893-5082698d-fd61-402a-80b9-fb15ae55dd25.jpg)

**Comment by Rektyfikowany** [comment]
在目录下添加`README.md`文件或[元信息](https://alist.nn.ci/zh/guide/advanced/meta.html)
